apps/simple/static/lite.html

```diff
@@ -11,14 +11,28 @@
         useBinder: false
       }
     </script>
-    <script id="jupyter-config-data" type="application/json">
-      {}
-    </script>
-    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <!-- <script id="jupyter-config-data" type="application/json">
+      {
+        "litePluginSettings": {
+          "@jupyterlite/pyodide-kernel-extension:kernel": {
+            "pipliteUrls": ["https://unpkg.com/@jupyterlite/pyodide-kernel@0.0.6/pypi/all.json"]
+          }
+        }
+      }
+    </script> -->
```

Comment:
This option could / should work as er make a call to `PageConfig.getOptions('')` on starting the server?

Reply:
Right the kernel should be able to pick it up here: https://github.com/jupyterlite/pyodide-kernel/blob/01ef2f5ad3c468205d78dddeffa2990c0bd4eee9/packages/pyodide-kernel-extension/src/index.ts#L45